### DISCUSSION TITLE
CFBW Engine interior sounds are absolutely amazing

### DISCUSSION THREAD

**Discussion by Tibu73-ui** [flybywiresim/aircraft#3215] [General]
I want to agree new sounds added for A32nx almost in Custom version. Interior (cabin and pax views engine sounds are really well done, as well APU bleed sound is fantastic, but I don't like so much the APU sound in external view.
Congrats to the team sound designer, don't change these sounds, are really amazing.

**Comment by hotshotp** [comment]
thank you! maybe you don't like the exterior apu bleed sound but that is how it is in real life, it is very loud, louder than apu

**Comment by Tibu73-ui** [comment]
Yeah, agree with you but when is engaged and fully working , its a fix
point center pan. When move the cámara in exterior view around the aircraft
the sound don't pan left right and change some volume and for me its a
little bit unrealistic the sound Fx.

**Reply by hotshotp** [reply]
the sound is directional and youre the only one who seems to have the issue actually

**Comment by Tibu73-ui** [comment]
With last  versionall is fine.
You are doing an amazing job.
Congrats!

**Reply by hotshotp** [reply]
Awesome! Thanks for your feedback
